src/retrace/retrace.py

```diff
@@ -1653,7 +1653,7 @@ def has_notify(self):
 
     def get_notify(self):
         result = self.get(RetraceTask.NOTIFY_FILE, maxlen=1 << 16)
-        return filter(None, set(n.strip() for n in result.split("\n")))
+        return [_f for _f in set(n.strip() for n in result.split("\n")) if _f]
```

Comment:
The `get_notify` loads e-mails (for notifications) from file.

Can you change the `_f` to `mail` or `email`?